osquery_rules/osquery_mac_osx_attacks_keyboard_events.py

```diff
@@ -16,17 +17,17 @@
 
 
 def rule(event):
-    if 'Keyboard_Event_Taps' not in event['name']:
+    if 'Keyboard_Event_Taps' not in event.get('name', ''):
         return False
 
-    if event['action'] != 'added':
+    if event.get('action') != 'added':
         return False
 
-    process_path = event['columns'].get('path')
-    if not process_path:
+    process_path = deep_get(event, 'columns', 'path', default='')
+    if process_path == '':
```

Comment:
I think we could also leave it with the None check if we left the default out here - just another option

Reply:
We can't leave the default here because if we pass in None to `fnmatch` in line 38, we get the same issue